generators/app/templates/src/screens/Dashboard/screens/Home/index.test.tsx

```diff
@@ -0,0 +1,72 @@
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import { actionCreators as authActions, ActionTypes, ResetUser } from 'contexts/UserContext/reducer';
+import { withContextProvider as withUserContext } from 'contexts/UserContext';
+import * as AuthService from 'services/AuthService';
+
+import { withContextProvider as withTechContext } from './context';
+
+import Home from '.';
```

Comment:
```suggestion
import Home from '.';

const HomeWithUserContext = withUserContext(Home);
```

You can define the const here and use it in the tests.

I'm not sure if we should use the `withUserContext` since this a HomeTest. You could mock the context.